src/main/scala/scala/tools/refactoring/implementations/oimports/Region.scala

```diff
@@ -69,7 +69,7 @@ object Region {
 
   private def cutPrefixSuffix(imp: Global#Import, source: SourceFile): (String, List[String]) = {
     val printedImport = source.content.slice(imp.pos.start, imp.pos.end).mkString
-    val prefixPatternWithCommentInside = """import (((\/\*.*\*\/)*(\w|\d|_|-)+(\/\*.*\*\/)*)\.)+(\/\*.*\*\/)*""".r
+    val prefixPatternWithCommentInside = """import (((\/\*.*\*\/)*((\w|\d|_|-)+|(\`.*\`)+)(\/\*.*\*\/)*)\.)+(\/\*.*\*\/)*""".r
```

Comment:
As for the regular expression: Try (note the extra space after `import`) organizing imports here:

```scala
class Bug4 {
  import  java.util.ArrayList

  val x: ArrayList[Int] = ???
}
```

Maybe using `SourceWithMarker` might make your life easier and the code more readable?